drizzle-orm/src/sql/sql.ts

```diff
@@ -319,6 +319,16 @@ export class SQL<T = unknown> implements SQLWrapper {
 		this.shouldInlineParams = true;
 		return this;
 	}
+
+	/**
+	 * This method is used to conditionally include a part of the query.
+	 *
+	 * @param condition - Condition to check
+	 * @returns SQL if the condition is true, otherwise undefined
+	 */
+	if(condition: any): this | undefined {
```

Comment:
```suggestion
	if(condition: boolean): this | undefined {
```

Reply:
nope, based on discussion we've decided to comply with `if()` operator